adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -63,22 +108,45 @@ + (GADVersionNumber)adapterVersion {
 
 - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
                    completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+    [BUAdSDKManager setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    _bannerRenderer = [[GADPangleRTBBannerRenderer alloc]init];
+    [_bannerRenderer renderBannerForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadInterstitialForAdConfiguration:(GADMediationInterstitialAdConfiguration *)adConfiguration
                          completionHandler:(GADMediationInterstitialLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding interstitial ad.
-}
-
-- (void)loadNativeAdForAdConfiguration:(GADMediationNativeAdConfiguration *)adConfiguration
-                     completionHandler:(GADMediationNativeLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding native ad.
+    [BUAdSDKManager setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    _interstitialRenderer = [[GADPangleRTBInterstitialRenderer alloc]init];
+    [_interstitialRenderer renderInterstitialForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
                        completionHandler:(GADMediationRewardedLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding rewarded ad.
+    [BUAdSDKManager setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    _rewardedlRenderer = [[GADPangleRTBRewardedRenderer alloc]init];
+    [_rewardedlRenderer renderRewardedAdForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
++ (void)setCoppa:(NSInteger)coppa {
+    _coppa = coppa;
```

Comment:
Would prefer validating the coppa value before doing this, just in case the publishers input invalid values (e.g. 100).

The same applies to `setGDPR` and `setCCPA` below.